src/renderer/abstractions/IProjectService.ts

```diff
@@ -134,7 +134,8 @@ export type IProjectService = {
    * @param file File name
    * @param contents File contents to save
    */
-  saveFileContent(file: string, contents: string | Uint8Array, asYouType?: boolean): Promise<void>;
+  saveFileContent(file: string, contents: string | Uint8Array): Promise<void>;
+  saveFileContentAsYouType(file: string, contents: string | Uint8Array): Promise<void>;
```

Comment:
I came to an understanding the it is better to have here the two methods, after all.